src/Components/App.tsx

```diff
@@ -34,7 +34,7 @@ const App: FC = () => {
   return (
     <main>
       <Nav monsterViewHandler={() => monsterViewHandler()}/>
-      {(!monsterView) && <MainSection />}
+      {(!monsterView) && <CharacterSection />}
```

Comment:
I like the name character section for it more, good change!

Reply:
Yes it just makes more sense! Thanks!